var/spack/repos/builtin/packages/opengl/package.py

```diff
@@ -91,5 +91,34 @@ def fetcher(self):
 
     @property
     def libs(self):
```

Comment:
Restructure in a similar way to the mes `def libs(self):` function, including with the properties